webextension/scripts/utils.js

```diff
@@ -443,19 +443,27 @@ function opener(url, option, callback) {
       h = Math.floor(screen.height * 0.9)
     }
     chrome.windows.create({ url: url, width: w, height: h, top: 0, left: 0, type: 'popup' }, (window) => {
-      if (callback) { callback(window.tabs[0].id) }
+      if (callbackFn) { callbackFn(window.tabs[0].id) }
     })
   }
 }
 
-function notify(message, callback) {
+function notify(message, callbackFn) {
   let options = {
     type: 'basic',
     title: 'WayBack Machine',
     message: message,
     iconUrl: chrome.runtime.getURL('images/app-icon/app-icon96.png')
   }
-  chrome.notifications && chrome.notifications.create(options, callback)
+  chrome.notifications && chrome.notifications.create(options, callbackFn)
+}
+
+function callback() {
```

Comment:
Let's call this function something more descriptive, such as `errorLog`. In this case, no need to rename existing functions

Reply:
Renamed it to `checkLastError`